### PR TITLE
Enable MANAGE permission on startup.

### PR DESCRIPTION
**Pull request by aHenryJard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+Enable Jenkins.MANAGE permission to be used with an authorization strategy (Matrix or RBAC for example).
 
 Reference: [jep-223](https://github.com/jenkinsci/jep/tree/master/jep/223)
+
+## How to compile
+
+The Jenkins.MANAGE permission is annotated as beta, you need to use the beta flag to compile it.
+
+```
+mvn clean install -DuseBeta=true
```
Comment (amuniz) [inline README.md:12]: This should go into the pom as a property.

File: pom.xml [hunk 2]
```diff
@@ -22,6 +22,30 @@
             <url>https://opensource.org/licenses/MIT</url>
         </license>
     </licenses>
+    <!--TODO the goal is to use README as plugin documentation, url to be validated once in jenkinci organization-->
+    <url>https://github.com/jenkinsci/manage-permission</url>
```
Comment (amuniz) [inline pom.xml:26]:
```suggestion
    <url>https://github.com/jenkinsci/manage-permission-plugin</url>
```

File: src/main/java/io/jenkins/plugins/managepermission/ManagePermissionEnabler.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import hudson.init.Initializer;
+import jenkins.model.Jenkins;
+
+@Extension
+@Restricted(NoExternalUse.class)
+public class ManagePermissionEnabler {
+
+    @Initializer(after = InitMilestone.PLUGINS_STARTED)
+    public static void enableManagePermission() {
+        if (System.getProperty("jenkins.security.ManagePermission") == null) {
```
Comment (amuniz) [inline src/main/java/io/jenkins/plugins/managepermission/ManagePermissionEnabler.java:17]: Need to check if it's `true` instead of `null`

**Comment by aHenryJard** [comment]
see https://github.com/aHenryJard/manage-permission-plugin/pull/2

**Pull request closed without merging**